packages/front-end/components/Features/RuleList.tsx

```diff
@@ -95,7 +95,10 @@ export default function RuleList({
         setActiveId(active.id);
       }}
     >
-      <SortableContext items={items} strategy={verticalListSortingStrategy}>
+      <SortableContext
+        items={items.map((item) => item.id)}
+        strategy={verticalListSortingStrategy}
+      >
         {items.map(({ ...rule }, i) => (
```

Comment:
@Jeremy I am not 100% sure how this worked in the first place but it started complaining after I changed `feature.d.ts` thought I would point this out.